### PR TITLE
[danger] Generate PR only if the changelog section is present

### PR DESCRIPTION
**Pull request by lukmccall**
# Why

Generating PR only if the changelog section is present.

# Test

On this PR, see:  https://github.com/expo/expo/pull/8533#issuecomment-635263820

# Changelog
- [expo-permissions] Test
- [expo-file-system]

### CODE REVIEW

**Comment by lukmccall** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  
📋 **Missing Changelog**
------
🛠 Add missing entires to:
- <code><a href="https://github.com/expo/expo/blob/@lukmccall/danger/add-skip-option/packages/expo-permissions/CHANGELOG.md">packages/expo-permissions/CHANGELOG.md</a></code>
  </td>
    </tr>
  </tbody>
</table>




### 🛠 Suggested fixes:

<details>
  <summary>📋 Missing changelog</summary>

  #### Apply suggested changes:
```diff
diff --git a/packages/expo-permissions/CHANGELOG.md b/packages/expo-permissions/CHANGELOG.md
index a3418909e3..6e1a3c5ffe 100644
--- a/packages/expo-permissions/CHANGELOG.md
+++ b/packages/expo-permissions/CHANGELOG.md
@@ -10,6 +10,8 @@
 
 ### 🐛 Bug fixes
 
+- Add skip option ([#8533](https://github.com/expo/expo/pull/8533) by [@lukmccall](https://github.com/lukmccall))
+
 ## 8.2.0 — 2020-05-27
 
 ### 🐛 Bug fixes
```

#### or merge this pull request: https://github.com/expo/expo/pull/8534 
</details>
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 83168079b82f4c4f47badac571467fe3da1a37da
</p>

**Comment by lukmccall** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Changelog check for expo-permissions was skipped.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 83168079b82f4c4f47badac571467fe3da1a37da
</p>

File: packages/danger/src/PullRequestManager.ts [hunk 1]
```diff
@@ -59,63 +63,48 @@ export class PullRequestManager {
...
+        type: ChangelogEntryType.BUG_FIXES,
+        message: this.prTitle.replace(/\[.*\]/, '').trim(),
       },
     };
 
-    const parseLine: (line: string) => void = line => {
-      const parsingResult = this.parseTagsFromLine(line);
+    this.changelogSection.forEach(line => {
+      const { packageName, type } = this.parseTagsFromLine(line);
       const message = line.replace(/\[.*\]/, '').trim();
```
Comment (tsapeta) [inline packages/danger/src/PullRequestManager.ts:73]: Hmm, shouldn't it be 👇 ?
```suggestion
      const message = line.replace(/^(\[[^\]]\])+/g, '').trim();
```

**Review by tsapeta — APPROVED** [review]
Looks good, thanks 👍 Let's try it out 😄

**Pull request merged**